build_tfm.py

```diff
@@ -454,11 +442,41 @@ def _copy_library(source, toolchain):
     if not isdir(regression_lib_dest):
         os.makedirs(regression_lib_dest)
     shutil.copy2(
-        join(regression_lib_src, "libtfm_non_secure_tests.a"),
+        join(regression_lib_src, dest_lib_name),
```

Comment:
The file generated by TF-M's build system is always `libpsa_api_ns.a` for both toolchains. But as this PR isn't intended to cover regression tests yet, you can either fix it now or in the future.

Reply:
It's ok to fix it now...  as I saw a logical issue there

Reply:
So, is this going to be fixed in this PR after all?

Reply:
yes

Reply:
This still needs to be fixed,
```suggestion
        join(regression_lib_src, "libpsa_api_ns.a"),
```
The source lib is always the same, only the dest lib depends on the toolchain

Reply:
Ah ok, It thought it was `.ar`. I'm going to revert this change, anyways this is related to regression tests.